pkg/logging/interface.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package logging
+
+import (
```

Comment:
Maybe `options` file instead? (: As in grpc middelware

Reply:
Done